src/lib/components/copyInput.svelte

```diff
@@ -24,7 +24,7 @@
     };
 </script>
 
-<div>
+<div id="tooltip-container">
```

Comment:
It's best to avoid IDs when possible since they're highly selective regarding CSS, and IDs should be unique, but this component can be used multiple times/places. For example,

https://github.com/appwrite/console/blob/ffb7476b28e55b8fbffb57626bfa4f8cdbdaf821/src/routes/console/project-%5Bproject%5D/settings/updateName.svelte#L48-L49